contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -22,21 +22,24 @@ contract AaveStrategy is InitializableAbstractStrategy {
     IAaveIncentivesController public incentivesController;
     IAaveStakedToken public stkAave;
 
+    /**
+     * @param _stratConfig The platform and OToken vault addresses
+     */
+    constructor(BaseStrategyConfig memory _stratConfig)
```

Comment:
should all these constructor params be calldata instead of memory?

Reply:
only `storage` or `memory` is supported for struct param.